src/makeInfinityAdder.js

```diff
@@ -7,6 +7,22 @@
 
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const adder = (number) => {
+    if (number === undefined) {
+      const summedUp = sum;
+
+      sum = 0;
+
+      return summedUp;
+    }
+    sum += number;
```

Comment:
always add blank line after `if` statement
```suggestion
    }

    sum += number;
```